.github/workflows/continuous-integration.yml

```diff
@@ -380,6 +380,12 @@ jobs:
     name: 'Tests: End-to-end'
     runs-on: ubuntu-latest
     timeout-minutes: 30
+    strategy:
+      matrix:
+        browser:
+          - chrome
+          - firefox
+          - edge
```

Comment:
Also add electron, because that is the default browser when running e2e tests locally.